### PR TITLE
Config split base settings (carried over from GitLab)

### PR DESCRIPTION
**Pull request by johangant**

### CODE REVIEW

File: config/drupal.settings [hunk 1]
```diff
@@ -32,6 +32,11 @@ $config_directories[CONFIG_SYNC_DIRECTORY] = getenv('CONFIG_SYNC_DIRECTORY');
 // $settings['cache']['default'] = 'cache.backend.memcache';
 $settings['memcache']['servers'] = [sprintf('%s:%s', getenv('MEMCACHE_HOSTNAME'), getenv('MEMCACHE_PORT')) => 'default'];
 
+// Set config split environment.
+$config['config_split.config_split.local_development']['status'] = TRUE;
+$config['config_split.config_split.development']['status'] = FALSE;
```
Comment (omahm) [inline config/drupal.settings:37]: What's the difference between local_development and development ?

Comment (johangant) [reply config/drupal.settings:37]: local_development is effectively your local environment (Lando). I'd normally expect to call this 'dev' but that crosses over uncomfortably close to what is called the dev server (lives in the stables network) so would easily get confusing.

Happy to shorten to `local` if clearer?

Comment (omahm) [reply config/drupal.settings:37]: Yeah, local is clearer to me.

**Comment by DuttonMa** [comment]
LGTM

File: config/drupal.settings (hunk 1 above)
Comment (johangant) [reply config/drupal.settings:37]: Now done, cheers

**Pull request merged**